include/exadg/postprocessor/pointwise_output_generator_base.cpp

```diff
@@ -164,26 +164,25 @@ PointwiseOutputGeneratorBase<dim, Number>::setup_base(
 
 template<int dim, typename Number>
 void
-PointwiseOutputGeneratorBase<dim, Number>::add_scalar_quantity_dataset(std::string const & name)
+PointwiseOutputGeneratorBase<dim, Number>::add_quantity(std::string const & name,
+                                                        unsigned int const  n_components = 1)
```

Comment:
I think I would remove the default argument here (just following the standard C++ recommendation to avoid default arguments).

Reply:
[c697ee0](https://github.com/exadg/exadg/pull/255/commits/c697ee086e8bab1c98563b4d9219bf911538c4f9)